src/Microsoft.JSInterop/DotNetDispatcher.cs

```diff
@@ -129,6 +129,51 @@ private static object InvokeSynchronously(string assemblyName, string methodIden
         public static void EndInvoke(long asyncHandle, bool succeeded, object resultOrException)
             => ((JSRuntimeBase)JSRuntime.Current).EndInvokeJS(asyncHandle, succeeded, resultOrException);
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="dotNetObjectId"></param>
+        /// <param name="methodIdentifier"></param>
+        /// <param name="argsJson"></param>
+        /// <returns></returns>
```

Comment:
Needs actual docs 😆

Reply:
Yes, and tests! I only had an hour to put this PR together earlier and wanted it to show up before you started your work today.